### DISCUSSION TITLE
Which version of Ghostscript is supported with ImageMagick 7.1.1-11

### DISCUSSION THREAD

**Discussion by ptiruvaipati** [ImageMagick/ImageMagick#6395] [Development]
### ImageMagick version

7.1.1-11

### Operating system, version and so on

Windows

### Description

Hi,

Can someone please tell me which version of Ghostscript is supported with ImageMagick 7.1.1-11 ?

Currently we are using Ghostscript 9.56 with ImageMagick 7.0.8-50. 

Regards,
Pavan

**Comment by fmw42** [comment]
Any version is supported. But some work better than others due to bugs/errors in Ghostscript.  Get the latest version that you can use.  I am using 9.56.1 and it seems to work fine.

**Comment by botsmack** [comment]
For what it's worth, there is a critical security vulnerability (CVE-2023-28879) in all Ghostscript versions prior to 10.01.0. It was fixed in 10.01.1, which was published on 2023-03-27: https://artifex.com/news/critical-security-vulnerability-fixed-in-ghostscript

https://nvd.nist.gov/vuln/detail/CVE-2023-28879

https://git.ghostscript.com/?p=ghostpdl.git;h=37ed5022cecd584de868933b5b60da2e995b3179

**Reply by fmw42** [reply]
Thanks.  There have been other previous versions that have bugs/issues. But any version works with Imagemagick. The GS vulnerabilities and bugs are another matter and yet still important.
